apps/src/netsim/NetSimLobby.js

```diff
@@ -206,6 +208,12 @@ NetSimLobby.prototype.render = function () {
   } else {
     // Node selection panel: The lobby list of who we can connect to, and
     // controls for picking one and connecting.
+    const selectedShard = _.find(
+      this.shardChoices_,
+      s => s.shardID === this.selectedShardID_
+    );
+    const sectionID = selectedShard ? selectedShard.shardSeed : null;
```

Comment:
We might be able to remove this if we need to track the sectionID at a higher level.